.github/workflows/accuracy-tests.yml

```
@@ -0,0 +1,48 @@
+name: Accuracy Tests
+
+on:
+  workflow_dispatch:
+  pull_request:
+    types: [labeled]
+
+jobs:
+  run-accuracy-tests:
+    name: Run Accuracy Tests
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: write
+    if: |
+      github.event_name == 'workflow_dispatch' ||
+      (github.event_name == 'pull_request' && github.event.label.name == 'accuracy-tests')
+    env:
+      MDB_OPEN_AI_API_KEY: ${{ secrets.MDB_OPEN_AI_API_KEY }}
+      MDB_GEMINI_API_KEY: ${{ secrets.MDB_GEMINI_API_KEY }}
+      MDB_AZURE_OPEN_AI_API_KEY: ${{ secrets.MDB_AZURE_OPEN_AI_API_KEY }}
+      MDB_AZURE_OPEN_AI_API_URL: ${{ secrets.MDB_AZURE_OPEN_AI_API_URL }}
+      MDB_ACCURACY_MDB_URL: ${{ secrets.MDB_ACCURACY_MDB_URL }}
```

```suggestion
      MDB_ACCURACY_MDB_URL: ${{ secrets.ACCURACY_MDB_CONNECTION_STRING }}
```

Also, we may want to propagate the connection string wording.